packages/example-project/component-library-react/src/react-component-lib/createComponent.tsx

```diff
@@ -24,14 +25,16 @@ export const createReactComponent = <
   ExpandedPropsTypes = {}
 >(
   tagName: string,
+  customElement?: any,
```

Comment:
Although this is optional, is there any concern that adding it in a place other than the end of the function signature will break existing clients? Or are we planning on making this a breaking change?

Reply:
Hmm yes that is true - this would cause a breaking change. The alternative is a bit uglier since you'll need to do `createReactComponent('ion-badge', undefined, undefined, IonBadgeCmp)` but that's probably the correct approach if we are looking to avoid breaking changes.